astropy/table/tests/test_serialize.py

```diff
@@ -0,0 +1,375 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
This file is factored out from the original `test_ecsv.py`. The goal is to unify testing for ECSV, FITS, and HDF5, but it's not qiute there yet. I point out below what is new.